tests/sentry/api/endpoints/test_organization_dashboard_details.py

```diff
@@ -320,6 +320,33 @@ def test_dashboard_widget_default_dataset_source_is_unknown(self):
         assert response.status_code == 200, response.content
         assert response.data["widgets"][0]["datasetSource"] == "unknown"
 
+    def test_dashboard_widget_query_returns_selected_aggregate(self):
```

Comment:
Can we also port these tests over to `test_organization_dashboards.py`?

Reply:
moved the dashboard level test (test_dashboard_widget_query_returns_selected_aggregate)